entities/statistics.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Statistics
+  include Validation
+
+  def top_reader_stats(orders, quantity)
+    orders.group_by(&:reader).sort_by { |_key, value| value.count }.reverse.first(quantity).to_h.keys
```

Comment:
move orders.group_by(&:reader).sort_by { |_key, value| value.count } to private separate method, and pass symbol to it.